src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumbsSearch.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import TextField from 'app/components/forms/textField';
-import {IconSearch} from 'app/icons/iconSearch';
+import IconSearch from 'app/icons/iconSearch';
 import {t} from 'app/locale';
-import {IconClose} from 'app/icons/iconClose';
+import IconClose from 'app/icons/iconClose';
```

Comment:
Should we change all of these icon imports (now that we are touching those files) to 
```
import {IconClose, IconSearch} from 'app/icons';
```
?

Reply:
@matejminar after reading this blog post https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/ I understand why you prefer named exports imports over default export and..I agree with your..let's do it! 🤜🤛